SimTKcommon/include/SimTKcommon/internal/Pathname.h

```diff
@@ -87,6 +87,26 @@ namespace SimTK {
  */
 class SimTK_SimTKCOMMON_EXPORT Pathname {
 public:
+    /// Given a specified working directory (swd) and path, this function
+    /// evaluates the absolute path of a given path relative to the swd and
+    /// returns the directory, fileName, and extension of the canonicalized absolute
+    /// path. This means that instead of evaluating "." as the current working
+    /// directory (cwd), the swd is used. In general, some rules are as follows
+    /// 1) If an absolute path is given by path, swd is not used. An absolute path
+    ///    is defined as a a root-relative path name and on Windows it will begin 
```

Comment:
"and on Windows it *may* begin with a drive letter" (it is still root-relative if it begins with a slash, but it is the root of the current drive). I suppose if there were an swd with a drive letter it would grab that drive rather than the current one, right? But then ignore the rest of the swd?